src/swupd.h

```diff
@@ -398,6 +399,23 @@ typedef enum telem_prio_t {
 } telem_prio_t;
 extern void telemetry(telem_prio_t level, const char *class, const char *fmt, ...);
 
+/* fs_storage.c */
+
+/* bundle_result contains the information to print along with
+ * the internal relevancy score used to sort the output */
+struct bundle_result {
+	char bundle_name[BUNDLE_NAME_MAXLEN];
+	long topsize;
+	long size;
+	double score;
+	struct list *files;
+	struct list *includes;
+	bool is_tracked;
+	bool seen;
```

Comment:
bool is_tracked:1;
bool seen:1;

Reply:
What do you mean? I should initialize the variables? (...because I probably should have...)

Reply:
that's used to use just one bit for those values. Take a look at struct file for example. Not sure though if this is going to work with bool, because unsigned int is used in other cases. can you check that?